generators/client-2/templates/src/main/webapp/app/home/home.html

```diff
@@ -4,7 +4,7 @@
             <span class="hipster img-responsive img-rounded"></span>
         </div>
         <div class="col-md-8">
-            <h1 translate="home.title">Welcome, Java Hipster!</h1>
+            <h1 [jhi-translate]="'home.title' | translate">Welcome, Java Hipster!</h1>
```

Comment:
Ya that would be there with or without the pipe rite?
I would prefer a simpler directive which does everything. we can use the service inside the directive. Also I think its possible to get the directive evaluated during digest cycles